### DISCUSSION TITLE
Can a custom error page be defined for status code 502?

### DISCUSSION THREAD

**Discussion by domi-87** [Azure/application-gateway-kubernetes-ingress#1300] [Q&A]
In Azure, a specific error page for the status code "Bad gateway - 502" and "Forbidden - 403" can be defined for each listener under Application Gateway > Listeners. Can this also be specified via Helm Chart using an annotation or is there another way to define this via Helm Chart?

How does the Helm deployment behave when we define an Error page in Azure via GUI and make a new Helm deployment?

Thank you very much for the help. Unfortunately, we could not find a clue in the code or in the help forums/docs.

**Comment by Exchizz** [comment]
Any updates on this ?
